README.md

```
@@ -1,3 +1,318 @@
+## WhatsApp Message Flow Architecture
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)